### ISSUE TITLE
False positives are occurring with the no-dynamic-styling rule.

### ISSUE THREAD

**Issue by nusuke** [chakra-ui/eslint-plugin-panda#147]
My vite.config.ts, which should be unrelated, fails with a no-dynamic-styling rule.

An example.
```
import { defineConfig } from "vite";
import react from "@vitejs/plugin-react";

export default defineConfig({
  plugins: [react()],
});
```
Probably an error at this position in the code.
https://github.com/chakra-ui/eslint-plugin-panda/blob/main/plugin/src/rules/no-dynamic-styling.ts#L100-L103

Please review and confirm this issue.
